### ISSUE TITLE
Buy price for Crafting

### ISSUE THREAD

**Issue by Rixium** [LartTyler/MHWDB-Docs#27]
I can't see anything for this in the database. There is a crafting cost associated with items, that I don't have access to through this API. Could we get this functionality?

![image](https://user-images.githubusercontent.com/3577800/40056771-45c35714-5844-11e8-9f41-e46df362ac52.png)

**Comment by LartTyler** [comment]
Yep, I'll add this to the backlog. The current crafting info is pulled from Kiranico, which doesn't include costs, and I haven't had a chance to rewrite it to pull the missing info from another source. I'll keep you posted!

**Comment by Rixium** [comment]
Awesome, thanks!
